src/wrappers/SD_TC.cpp

```diff
@@ -245,3 +254,40 @@ void SD_TC::todaysDataFileName(char* path, int size) {
   snprintf_P(path, size, (PGM_P)F("%4i%02i%02i.csv"), now.year(), now.month(), now.day());
   COUT(path);
 }
+
+void SD_TC::updateRemoteFileSize() {
+  File file = open(remoteLogName, O_RDONLY);
+  if (file) {
+    remoteFileSize = file.size();
+    file.close();
+  } else {
+    remoteFileSize = 0;
+  }
+}
+
+/**
+ * @brief write to the appropriate "remote" file on the SD card
+ *
+ * @param line
+ */
+void SD_TC::writeToRemoteLog(const char* line) {
+#if defined(ARDUINO_CI_COMPILATION_MOCKS)
+  strncpy(mostRecentRemoteEntry, line, sizeof(mostRecentRemoteEntry));  // Flawfinder: ignore
+  mostRecentRemoteEntry[sizeof(mostRecentRemoteEntry) - 1] = '\0';      // Ensure null-terminated string
+#endif
+  if (!sd.exists(getRemoteLogName())) {
+    // rather than write an entire header line in one buffer, we break it into chunks to save memory
+    char buffer[200];
+    DataLogger::instance()->putRemoteFileHeader(buffer, sizeof(buffer), 0);
+    appendDataToPath(buffer, remoteLogName, false);
+    DataLogger::instance()->putRemoteFileHeader(buffer, sizeof(buffer), 1);
+    appendDataToPath(buffer, remoteLogName, false);
+    DataLogger::instance()->putRemoteFileHeader(buffer, sizeof(buffer), 2);
+    appendDataToPath(buffer, remoteLogName, false);
+    DataLogger::instance()->putRemoteFileHeader(buffer, sizeof(buffer), 3);
+    appendDataToPath(buffer, remoteLogName);
+  }
+  appendDataToPath(line, remoteLogName);
+  updateRemoteFileSize();
+  // AlertPusher::instance()->pushSoon();
```

Comment:
So this commented-out line is a placeholder for the next PR, right? It triggers the network process for transferring data from the local "remote" file to its copy on the OAP server, right?

Reply:
Yes, exactly...